tests/greendns_test.py

```diff
@@ -931,6 +931,23 @@ def test_noraise_dns_tcp(self):
             self.assertEqual(list(response.rrset.items)[0].address, expected_ip)
 
 
+class TestRaiseErrors(tests.LimitedTestCase):
+
+    def test_raise_new_error(self):
+        # https://github.com/eventlet/eventlet/issues/810
+        if not hasattr(greendns.EAI_EAGAIN_ERROR, "__traceback__"):
+            self.skipTest("Python version does not implement PEP-3134, skip this testcase.")
```

Comment:
We recently removed support of Python version lower than 3.8, so I think we can also start to remove this kind of version check.

Opinion?

Reply:
Yeah, if we only support python3.8+, this if expression will always be true, just remove it.

Reply:
We recently reintroduced support of Python 3.7 (https://github.com/eventlet/eventlet/issues/846), however, I don't think that will impact this code.